packages/time/README.md

```diff
@@ -3,7 +3,44 @@
 The plugin for [Size Limit] to track JS download and execution time by [estimo]
 and Puppeter.
 
-See Size Limit docs for more details.
+## Custom options
```

Comment:
Let’s replace it to `Customize Network Speed` (capitalized)

Reply:
Updated, thank you!